python/lsst/daf/persistence/test/testObject.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2016 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+
+class TestObject(object):
+    """A basic object for testing. Contains a data blob that can be compared with other TestObject instances
+    for equality (or inequality).
+    """
+
+    def __init__(self, data):
+        self.data = data
+
+    def __eq__(self, other):
+        return self.data == other.data
```

Comment:
Even if you don't use it, I'd recommend implementing `__ne__` as well, just so avoid confusing behavior down the road (if you don't implement it, `!=` will fall back to pointer equality instead of the inverse of your custom equality operator).